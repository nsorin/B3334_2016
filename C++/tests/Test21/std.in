R r1 5 10 10 5
R r2 7 20 20 7
PC pc1 5 5 5 10 7 20 20 20 20 7 10 5 5 5
OI r1int2 r1 r2
OI all r1 r2 pc1
HIT r1int2 8 9
HIT r1int2 8 6
HIT r1int2 8 12
HIT all 8 9
HIT all 8 6
HIT all 8 12
HIT pc1 11 6
HIT pc1 11 5
EXIT
