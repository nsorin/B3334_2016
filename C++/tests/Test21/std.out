#R r1 5 5 10 10
OK
#R r2 7 7 20 20
OK
#PC pc1 5 5 5 10 7 20 20 20 20 7 10 5 5 5
OK
#OI r1int2 r1 r2
OK
#OI all r1 r2 pc1
OK
#HIT r1int2 8 9
YES
#HIT r1int2 8 6
NO
#HIT r1int2 8 12
NO
#HIT all 8 9
YES
#HIT all 8 6
NO
#HIT all 8 12
NO
#HIT pc1 11 6
YES
#HIT pc1 11 5
NO
#EXIT
